src/Plugins/SimplnxCore/src/SimplnxCore/Filters/Algorithms/ComputeBoundingBoxStats.cpp

```diff
@@ -372,7 +372,14 @@ class ComputeBasicAndFrequencyStatsImpl
             count++;
             minValue = std::min(minValue, value);
             maxValue = std::max(maxValue, value);
-            summationValue += value;
+            if constexpr(std::is_same_v<T, bool>)
```

Comment:
Now that we've disallowed bool from the template we don't need these `if constexpr` blocks anymore. We could also add an additional check and add a concept to the template parameter of the struct to prevent bool from working.